_data-prepper/pipelines/configuration/sinks/s3.md

```diff
@@ -59,33 +93,34 @@ Option | Required | Type | Description
+
 
 
 ## Threshold configuration
 
-Use the following options to set ingestion thresholds for the `s3` sink.
+Use the following options to set ingestion thresholds for the `s3` sink. When any of these conditions is met, Data Prepper will write events to an S3 object.
 
 Option | Required | Type | Description
 :--- | :--- | :--- | :---
-`event_count` | Yes | Integer | The maximum number of events the S3 bucket can ingest. 
-`maximum_size` | Yes | String | The maximum number of bytes that the S3 bucket can ingest after compression. Defaults to `50mb`.
-`event_collect_timeout` | Yes | String | Sets the time period during which events are collected before ingestion. All values are strings that represent duration, either an ISO_8601 notation string, such as `PT20.345S`, or a simple notation, such as `60s` or `1500ms`.
+`event_count` | Yes | Integer | The number of Data Prepper events to accumulate before writing an object to S3.
+`maximum_size` | No | String | The maximum number of bytes to accumulate before writing an object to S3. Defaults to `50mb`.
+`event_collect_timeout` | Yes | String | The maximum timeout before Data Prepper writes an event to S3. The value should be either an ISO-8601 duration, such as `PT2M30S`, or a simple notation, such as `60s` or `1500ms`.
 
 
 ## Buffer type
 
-`buffer_type` is an optional configuration that records stored events temporarily before flushing them into an S3 bucket. The default value is `in_memory`. Use one of the following options: 
+`buffer_type` is an optional configuration that determines how Data Prepper temporarily holds data before writing an S3 object to S3. The default value is `in_memory`. Use one of the following options:
 
 - `in_memory`: Stores the record in memory.
-- `local_file`: Flushes the record into a file on your machine. 
+- `local_file`: Flushes the record into a file on your machine. This uses your local machine's temporary directory.
 - `multipart`: Writes using the [S3 multipart upload](https://docs.aws.amazon.com/AmazonS3/latest/userguide/mpuoverview.html). Every 10 MB is written as a part.
 
 ## Object key configuration
 
 Option | Required | Type | Description
 :--- | :--- | :--- | :---
-`path_prefix` | Yes | String | The S3 key prefix path to use. Accepts date-time formatting. For example, you can use `%{yyyy}/%{MM}/%{dd}/%{HH}/` to create hourly folders in S3. By default, events write to the root of the bucket. 
+`path_prefix` | No | String | The S3 key prefix path to use for objects written to S3. Accepts date-time formatting. For example, you can use `%{yyyy}/%{MM}/%{dd}/%{HH}/` to create hourly folders in S3. You should have this end with a `/`. By default, Data Prepper writes objects to the root of the S3 bucket.
```

Comment:
Third sentence: Let's revise to "The [noun] should end with a `/`."

Reply:
"The path prefix should end with a `/`."

Reply:
@natebower , See the comment above.